src/isar/apis/api.py

```diff
@@ -80,6 +90,26 @@ def _create_app(self) -> FastAPI:
 
         app.include_router(router=self._create_info_router())
 
+        @app.middleware("http")
+        async def middlewareOpencensus(request: Request, call_next):
+            tracer = Tracer(
+                exporter=AzureExporter(),
```

Comment:
```suggestion
                exporter=AzureExporter(connection_string=settings.APPLICATIONINSIGHTS_CONNECTION_STRING),
```

Reply:
This one also gets it automatically (like the log handler), should i set both explicitly or neither?

Reply:
Will set both explicitly